src/index.html

```diff
@@ -60,9 +60,34 @@ <h2>Daily Stats</h2>
       </article>
     </section>
 
-    <section class="graph">
```

Comment:
There's still formatting to be done on the "today's water consumption" but hey, we're getting numbers to show up!